Cubical/Algebra/CommMonoid/GrothendieckGroup.agda

```diff
@@ -74,13 +72,13 @@ module _ (M : CommMonoid ℓ) where
 
             -- remove proof when MonoidSolver is done
```

Comment:
Don't we have a monoid solver that can be used now ( https://github.com/agda/cubical/tree/master/Cubical/Algebra/MonoidSolver)? @felixwellen